src/main/java/org/jboss/tools/intellij/openshift/WindowToolFactory.java

```diff
@@ -46,7 +47,7 @@ public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindo
         tree.putClientProperty(Constants.STRUCTURE_PROPERTY, structure);
         tree.setCellRenderer(new NodeRenderer());
         tree.setRootVisible(false);
-        PopupHandler.installPopupHandler(tree, "org.jboss.tools.intellij.tree", ActionPlaces.UNKNOWN);
+        PopupHandler.installPopupMenu(tree, "org.jboss.tools.intellij.tree", ActionPlaces.MAIN_MENU);
```

Comment:
@sbouchet this fails to compile currently:
The method is only available in 212.4746.92 while our build here targets 212+
We have a compatibility wrapper in intellij-common to solve this (that I am using in intellij-kubernetes):
https://github.com/redhat-developer/intellij-common/blob/06f9ee4999320f9c542b8558ce2e1e53aab3340e/src/main/java/com/redhat/devtools/intellij/common/compat/PopupHandlerAdapter.java#L29

Reply:
done.